metricbeat/etc/kibana/dashboard/Metricbeat-Apache-HTTPD-server-status.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
Not sure why these were removed. Make sure to rebase on top of master and run `make collect`.